src/ActionsServiceProvider.php

```diff
@@ -29,9 +30,15 @@ public function boot()
      */
     public function register()
     {
+        // Facade binding.
         App::bind('actions', function () {
             return new Action();
         });
+
+        // Interface binding.
+        App::bind(Actionable::class, function () {
+            return new Action();
+        });
     }
```

Comment:
Just saw that you could bind things via `bindings` properties just curious to see what are your thoughts on using properties here or if they could be used?

Reply:
This is an excellent reminder. I also attempted to put the facade in there... it works!